docs/io.cozy.triggers.state.md

```diff
@@ -0,0 +1,19 @@
+- [Cozy triggers state doctype](#cozy-triggers-state-doctype)
+  - [`io.cozy.triggers.state`](#iocozytriggersstate)
+
+# Cozy triggers state doctype
+
+## `io.cozy.triggers.state`
```

Comment:
I think we should make it clear that this doctype is not persisted in CouchDB, its data is recomputed from jobs.